app/hooks/useLazyMint.tsx

```
@@ -0,0 +1,69 @@
+        if (!args[key]) {
+          // throw new Error(`${key} is required`);
+        }
+      }
+
+      const res = await fetch(baseURIForTokens);
+      const data = await res.json();
+      const tknMetadata = Object.assign(data, {
+        properties: {
+          amount,
+          price,
+          creatorAddress: activeAccount?.address!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)